src/effects/effectchainslot.cpp

```diff
@@ -68,6 +68,12 @@ EffectChainSlot::EffectChainSlot(EffectRack* pRack, const QString& group,
 
     connect(&m_channelStatusMapper, SIGNAL(mapped(const QString&)),
             this, SLOT(slotChannelStatusChanged(const QString&)));
+
+    // It is up to skins and controller mappings to do anything useful with this.
+    m_pControlExpanded = new ControlPushButton(ConfigKey(m_group, "expanded"));
```

Comment:
Okay, that makes sense. I can move it into each of the official skins. I think good documentation for skin authors would solve the issue (see also https://bugs.launchpad.net/mixxx/+bug/1647221 ).